MapboxCoreNavigation/SpokenInstructionFormatter.swift

```diff
@@ -76,7 +76,12 @@ public class SpokenInstructionFormatter: NSObject {
                 text = String.localizedStringWithFormat(NSLocalizedString("CONTINUE", bundle: .mapboxCoreNavigation, value: "Continue for %@", comment: "Format for speech string after completing a maneuver and starting a new step; 1 = distance"), escapeIfNecessary(maneuverVoiceDistanceFormatter.string(from: userDistance)))
             }
         } else if alertLevel == .high && upcomingStepDuration < linkedInstructionMultiplier {
-            text = String.localizedStringWithFormat(NSLocalizedString("LINKED_UTTERANCE_FORMAT", bundle: .mapboxCoreNavigation, value: "%@, then %@", comment: "Format for speech string; 1 = current instruction; 2 = the following linked instruction"), upComingInstruction, followOnInstruction)
+            // If the upcoming step is an .exitRoundabout or .exitRotary, don't link the instruction
+            if let upcomingStep = routeProgress.currentLegProgress.upComingStep, upcomingStep.maneuverType == .exitRoundabout || upcomingStep.maneuverType == .exitRotary {
```

Comment:
Why `upComingStep`? Wouldn’t the `upComingStep` be `takeRoundabout` and `followOnStep` be `exitRoundabout`?

Reply:
* If `upComingStep` is `takeRoundabout` and `followOnStep` is `exitRoundabout`, we announce `takeRoundabout` then `exitRoundabout`. 🙅‍♂️
* If `upComingStep` is `exitRoundabout` and `followOnStep` is `turn`, we announce just the `exitRoundabout` and never announce the `turn`. 🙅‍♂️